code/online_challenges/src/project_euler/problem_020/problem_020.java

```diff
@@ -0,0 +1,25 @@
+import java.math.BigInteger;
+
+public class problem_020 {
```

Comment:
Make sure you are using the correct naming conventions. (I believe it should be PascalCase)